scripts/install-thrift.sh

```diff
@@ -7,13 +7,22 @@ if [ -z "${1}" ]; then
   exit 1
 fi
 
-BIN_FILE="thrift-1"
-TAR_FILE="${BIN_FILE}-$(uname -s | tr '[:upper:]' '[:lower:]')-$(uname -m).tar.gz"
-TAR_LOCATION="https://github.com/uber/tchannel-go/releases/download/thrift-v1.0.0-dev/${TAR_FILE}"
```

Comment:
Previously, they made a release which contains the compiled thrift version on tchannel-go, and then they curl it on build. I changed this to store it into the git repo, which is usually non-ideal but might be okay here if no more development will occur to tchannel-go for Temporal.

Let me know what you think.